app/screens/BookDetail/components/BookDetailOptions/style.js

```diff
@@ -1,8 +1,8 @@
 import { StyleSheet } from 'react-native';
+import { hightSize, mediumSize } from '@scss/fonts';
+import { darkGray, white } from '@scss/colors';
 
```

Comment:
remove this empty line unless you get an error

Reply:
In this case generate an error because i have one `@utils` file (index)